lib/nerves_hub_link/health_check.ex

```diff
@@ -0,0 +1,49 @@
+defmodule NervesHubLink.HealthCheck do
```

Comment:
We can skip this file and add the smarts to the `Client` module instead.

Reply:
Will look into that :)